flink-autoscaler-standalone/src/main/java/org/apache/flink/autoscaler/standalone/config/AutoscalerStandaloneOptions.java

```diff
@@ -109,4 +110,51 @@ private static ConfigOptions.OptionBuilder autoscalerStandaloneConfig(String key
                                                     + "export the password using this environment variable.",
                                             code(STATE_STORE_TYPE.key()), code(JDBC.toString()))
                                     .build());
+
+    public static final ConfigOption<EventHandlerType> EVENT_HANDLER_TYPE =
+            autoscalerStandaloneConfig("event-handler.type")
+                    .enumType(EventHandlerType.class)
+                    .defaultValue(EventHandlerType.LOGGING)
+                    .withDescription("The autoscaler event handler type.");
```

Comment:
What if users want to configure both logging and storing via JDBC?

Reply:
We can configure them separately.

```
--autoscaler.standalone.state-store.type jdbc \
--autoscaler.standalone.event-handler.type jdbc \
```

After users choose JDBC for them, we can reuse all jdbc related options, such as:

```
autoscaler.standalone.jdbc.username
autoscaler.standalone.jdbc.url
```

Reply:
Got it. That makes sense. I meant what if I want to print both to the console and store in JDBC? Currently there can only be one event handler active at a time.

Reply:
Sorry, I misunderstand your question before, thanks for your clarification!

> I meant what if I want to print both to the console and store in JDBC? Currently there can only be one event handler active at a time.

You are right, currently there are only be one event handler or state store active at a time. I see kubernetes operator is similar, kubernetes operator only creates one event handler or state store.

If users want to use multiple event handlers, I can implement `CompositeEventHandler` in the future. Users can configure:

```
--autoscaler.standalone.event-handler.type composite
--autoscaler.standalone.event-handler.composite.types jdbc,logging
```

The `CompositeStateStore` is similar to `CompositeEventHandler`:

```
--autoscaler.standalone.state-store.type composite
--autoscaler.standalone.state-store.composite.types jdbc,memory
```

In the short term, it seems unnecessary. I can implement them in the future if some users feedback it's useful in some scenarios. What do you think about the solution and time plan?

Reply:
That sounds good to me. We can add the composite handler if necessary in the future.